flink-connectors/flink-hbase/src/test/java/org/apache/flink/addons/hbase/example/HBaseReadExample.java

```diff
@@ -86,8 +86,8 @@ public boolean filter(Tuple2<String, String> t) throws Exception {
 
 		hbaseDs.print();
 
-		// kick off execution.
-		env.execute();
+		// kick off execution is not needed.
+		// env.execute();
```

Comment:
I'd just remove it.

Reply:
Cool! So should I close this PR?

Reply:
you can add another commit that removes the code

Reply:
Done.